contents/founders/first-1000-users.md

```diff
@@ -0,0 +1,227 @@
+---
+date: 2024-06-20
+title: 'How we got our first 1,000 users'
+author:
+  - james-hawkins
+  - andy-vandervell
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/math_meme_8b0013533e.jpg
+featuredImageType: full
+tags:
+  - Growth
+crosspost:
+  - Blog
+  - Newsletter
+---
+
+> This post is the first in a series where I’ll break down how we reached important milestones at PostHog.
+> 
+> - **Part 1:** How we got our first 1,000 users **(You are here)**
+> - **Part 2:** How we grew from 1,000 to 10,000 users (coming soon)
+> - **Part 3:** How we scaled to 100,000 users (coming soon)
+
+One of my favorite weeks as CEO of PostHog was an [offsite in Aruba](https://posthog.com/blog/aruba-hackathon), but it wasn't for the reason you expect. 
+
+We chose Aruba because the team deserved to celebrate. We’d grown massively, so we increased the offsite budget and opted for a fancy hotel in the Caribbean as a reward for everyone’s hard work.
+
+But I felt a little guilty when we arrived. 
+
+The lobby had a boat in it. There was a private beach. You could drink infinite piña coladas. I knew we were spending $100k more than we needed to and, as a founder, a core part of my job is making sure we don't run out of money!
+
+As any founder should, though, I:
+
+1. Compartmentalized my feelings (they don't add to shareholder value)
+2. Kept refreshing our revenue dashboard (TikTok told me manifesting is real)
+
+![posthog aruba](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/aruba/aruba-hackathon-photos.jpeg)
+
+It worked. Within two days, our annual recurring revenue (ARR) had grown enough to pay for the entire trip. Suddenly, I didn't feel guilty at all.
+
+This clearly demonstrated to me the efficiency of our business model – a model that’s propelled us from 10 users to 100,000 in just over four years. Today, around 1,200 companies – ranging from hobbyists to Fortune 500s – sign up every week. 
```

Comment:
```suggestion
This clearly demonstrated to me the efficiency of our business model – a model that’s propelled us from 10 users to over 160,000 in just over four years. Today, around 1,600 companies – ranging from hobbyists to Fortune 500s – sign up every week. 
```

Reply:
Where are you pulling that 160k figure from? I couldn't get that from numbers in PH, but probably depends on how you config the insight.